tests/test_data/tabs_and_spaces.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Library    OperatingSystem
+Documentation	Suite documentation.
```

Comment:
Should be separated by 2 spaces at least

Reply:
The commit review shows it wrong, but it is actually a tab between "Documentation" and "Suite documentation" for testing purposes.